source-code/core2/src/fullExample.ts

```diff
@@ -74,24 +71,42 @@ const inlang = createInlang({
 })
 
 // --- CRUD ---
-inlang.messages.query.create({ id: "myMessageId", languageTag: "en", pattern: [{"type": "Text", value:  "Hello World" }]})
+inlang.messages.query.create({
```

Comment:
is it just me or does the word `query` look odd when doing a mutation?

Reply:
@samuelstroschein ?

`inlang.message.query.get()`
`inlang.message.mutation.create()`